### PR TITLE
Issue/#66-Settings-Page

### LINKED ISSUE #66: Create a Settings Page
### User Story: 

As a software developer, I want to create a settings page so that the users of the app can change their setting while using the calendar app

### Acceptance Criteria:

1) A UI settings page that fits the needs of the application
2)

### Why is this Feature Needed?
We need a settings page so that users have the ability to customize their details when using the application

### PR DESCRIPTION
**Pull request by ghost**
- [ ] The pull request is complete according to the following criteria:
  - [x] Acceptance criteria have been met
  - [x] The documentation is kept up-to-date
  - [ ] Comprehensive tests (if applicable) have been generated and all pass.
  - [x] The pull request describes the changes that have been made, and enough information is present in the description for any developer to understand what has changed
  - [ ] Commits have been squashed (or will be on merge).
  - [ ] The branch name is descriptive and follows the pull request title format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [x] The pull request title is of the following format : {issue/bug...}/(Issue Number) - Name of issue. E.g bug/30-Fix-Project
  - [x] The description uses [github syntax](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to link to the issue. E,g Resolves se701g2/Doto#{Number}
  - [x] At least two reviewers assigned. One of which must be the assigner of the issue.
  - [ ] If there are merge conflicts, run git rebase as opposed to git merge with master.

### CODE REVIEW

**Review by isabellelucy — DISMISSED** [review]

File: doto-frontend/src/components/pages/Settings/SettingsPage.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    border-radius: 79px 0px 0px 0px;
+    width: 95vw;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    background-color: #8d6cd9;
+    min-height: 80vh;
+    margin-top: 20vh;
+    margin-left: 5vw;
+}
```
Comment (baja156) [inline doto-frontend/src/components/pages/Settings/SettingsPage.css]: are these consistent with the login page?

File: doto-frontend/src/components/pages/Settings/SettingsPage.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                <div className="column-fields">
+                    <h2 style={{ marginLeft: "10vw", marginTop: "4vh", textAlign: "left" }}>Theme:</h2>
+                    <div>
+                        <ThemeProvider theme={theme1}>
+                            <Button id="color-palette" variant="contained" color="primary"></Button>
+                        </ThemeProvider>
+                        <ThemeProvider theme={theme1}>
+                            <Button id="color-palette" variant="contained" color="secondary"></Button>
+                        </ThemeProvider>
+                    </div>
```
Comment (baja156) [inline doto-frontend/src/components/pages/Settings/SettingsPage.js]: are the empty div's really necessary? line 71 and 78

**Review by BabraAjaz — CHANGES_REQUESTED** [review]
Great job man. Just a few tiny things

File: doto-frontend/src/components/pages/Settings/SettingsPage.css (hunk 1 above)
Comment (lakshyaoberoi) [reply doto-frontend/src/components/pages/Settings/SettingsPage.css]: I'd sent my .css file to @FocalChord to use the same properties. So the login page would be consistent with this.

File: doto-frontend/src/components/pages/Settings/SettingsPage.js (hunk 2 above)
Comment (lakshyaoberoi) [reply doto-frontend/src/components/pages/Settings/SettingsPage.js]: No. Just took them out.

**Review by FocalChord — APPROVED** [review]
LGTM!

**Review by rishab-goswami — APPROVED** [review]

**Review by BabraAjaz — APPROVED** [review]
LGMT (y)

**Pull request merged**